lib/clientsidescripts.js

```diff
@@ -676,20 +676,26 @@ functions.findByPartialButtonText = function(searchText, using) {
  * Find elements by css selector and textual content.
  *
  * @param {string} cssSelector The css selector to match.
- * @param {string} searchText The exact text to match.
+ * @param {string} searchText The exact text to match.  Can be a serialized regex.
```

Comment:
Would it be better if we say: searchText The exact text to match or a serialized regex.

Reply:
sure